src/jit/flowgraph.cpp

```diff
@@ -21721,6 +21742,58 @@ void Compiler::fgDebugCheckBlockLinks()
     }
 }
 
+//------------------------------------------------------------------------------
+// fgDebugCheckNodesUniqueness: Check that each tree in the method has its own unique gtTreeId.
+//
+void Compiler::fgDebugCheckNodesUniqueness()
+{
+    unsigned     numberOfTrees = JitTls::GetCompiler()->compGenTreeID;
+    BitVecTraits nodesVecTraits(numberOfTrees, this);
+    BitVec       uniqueNodes(BitVecOps::MakeEmpty(&nodesVecTraits));
+
+    ArrayStack<unsigned> stack(this);
+
+    for (BasicBlock* block = fgFirstBB; block; block = block->bbNext)
```

Comment:
I think the recommendation is to use `block != nullptr` rather than just `block`.

Reply:
Fixed